test/models/movie.test.js

```diff
@@ -13,10 +16,29 @@ describe('movie model', () => {
         'id',
         'title',
         'release_year',
+        'locations',
         'object'
       ]);
     });
 
+    it('serialzes with location', async () => {
+      const movie_payload = { name: 'Zodiac' };
+      const location_payload = { name: 'Bay Bridge' };
+
+      await Knex.raw('TRUNCATE movies CASCADE; TRUNCATE locations CASCADE; TRUNCATE locations_movies CASCADE;');
+
+      const movie = await new Movie().save(movie_payload);
+      const location = await new Location().save(location_payload);
+      const loc = await new Location({ id: location.id }).fetch();
+
+      await new Movie({ id: movie.id }).locations().attach(loc);
+
+      const movie_fetch = await new Movie({ id: movie.id }).fetch({ withRelated: ['locations'] });
```

Comment:
Make sure to actually call `serialize` if we want to test it! Right now this test looks more like a controller test, so I think an easier way to test for serialization in this case would be to 
1) Create a movie model
2) Create a location model
3) Set the movie's location attribute to the location (e.g. `movie.locations = location). 
4) Call `movie.serialize()` 
5) Verify that the serialized model has the correct location.

but lmk what you think!

Reply:
i changed this test a little bit. good call on not needing `loc`. got away with using the `movie` and `location` variables from before. i do have to call one `.fetch`. This is to get the related locations using `withRelated`. I then serialize and check the fields. Let me know if this is more what you were thinking.